framework/doc/packages_config.yml

```diff
@@ -4,10 +4,14 @@ minimum_gcc: 7.5.0
 minimum_clang: 10.0.1
 minimum_python: 3.7
 mpich: 4.0.2
+openmpi: 4.1.6
 petsc_alt: 3.11.4
 petsc: 3.20.3
 moose_dev: 2024.01.23
 moose_tools: 2023.12.20
 apptainer_rockylinux: 8.8.20230518
 apptainer_mpich: 3.4.3
 apptainer_gcc: 12.2.1
+conda_version: 23.11.0-0
+conda_mpi: moose-mpi
+mpi: openmpi
```

Comment:
what s the difference between those 2

Reply:
This will control the default MPI wrapper that gets installed when multiple variants become available: https://github.com/idaholab/moose/pull/26848. The idea is:

MPICH variant
```bash
conda create -n compilers moose-mpi mpich
```

OpenMPI variant
```bash
conda create -n compilers moose-mpi openmpi
```

All contained in one recipe (moose-mpi). This allows a user to throw 'mpich' or 'openmpi' anywhere among their install line, and it will do the right thing:
```bash
conda create -n moose moose-dev openmpi
```